src/strategies/DeltaNeutralLp.sol

```diff
@@ -17,65 +17,81 @@ import {AccessStrategy} from "./AccessStrategy.sol";
 import {IMasterChef} from "src/interfaces/sushiswap/IMasterChef.sol";
 import {SlippageUtils} from "src/libs/SlippageUtils.sol";
 
+struct LenderInfo {
+    ILendingPool pool; // lending pool
+    ERC20 borrow; // borrowing asset
+    AggregatorV3Interface priceFeed; // borrow asset price feed
+}
+
+struct LpInfo {
+    IUniswapV2Router02 router; // lp router
+    IMasterChef masterChef; // gov pool
+    uint256 masterChefPid; // pool id
+    bool useMasterChefV2; // if we are using MasterChef v2
+    ERC20 sushiToken; // sushi token address, received as reward
+}
```

Comment:
It would be good to use NatSpec here. You can do this later (make an issue if you are punting).